### PR TITLE
update DE tranlation

### PR DESCRIPTION
**Pull request by mxr0**
update DE translation, Fix variable typo, improve/unify translation of 'event'

### CODE REVIEW

File: config/locales/de/cms.yml [hunk 1]
```diff
@@ -99,14 +99,14 @@ de:
         notifications: "Benachrichtigungseinstellungen"
 
     hints:
-      unnamed: "(Unnamed)" # TODO: Translate
+      unnamed: "(ohne Name)"
```
Comment (Ardnived) [inline config/locales/de/cms.yml:102]: This should probably be capitalised.

File: config/locales/de/models.yml [hunk 2]
```diff
@@ -76,13 +76,13 @@ de:
...
         # Basic pluralization
-        single: "Responsibility"
-        plural: "Responsibilities"
+        single: "Verantwortung"
+        plural: "Verantwortlichkeiten"
         # Countable
-        one: "responsibility"
-        other: "responsibilities"
+        one: "Verantwortung"
+        other: "Verantwortlichkeiten"
```
Comment (Ardnived) [inline config/locales/de/models.yml:85]: These should probably not be capitalised, as they will be used in the middle of a sentence.

File: config/locales/de/models.yml [hunk 3]
```diff
@@ -41,25 +41,25 @@ de:
...
-        single: "Online Event"
-        plural: "Online Events"
+        single: "Online Veranstaltung"
+        plural: "Online Veranstaltungen"
         # Countable
-        one: "online event"
-        other: "online events"
-      offline_event: # TODO: Translate
+        one: "Online Veranstaltung"
+        other: "Online Veranstaltungen"
```
Comment (Ardnived) [inline config/locales/de/models.yml:55]: These should probably not be capitalised, as they will be used in the middle of a sentence.

File: config/locales/de/models.yml [hunk 4]
```diff
@@ -41,25 +41,25 @@ de:
...
         # Basic pluralization
-        single: "In-Person Event"
-        plural: "In-Person Events"
+        single: "Vorort Veranstaltung"
+        plural: "Vorort Veranstaltungen"
         # Countable
-        one: "in-person event"
-        other: "in-person events"
+        one: "Vorort Veranstaltung"
+        other: "Vorort Veranstaltungen"
```
Comment (Ardnived) [inline config/locales/de/models.yml:62]: These should probably not be capitalised, as they will be used in the middle of a sentence.

File: config/locales/de/models.yml [hunk 5]
```diff
@@ -223,19 +223,19 @@ de:
...
           facebook: "Facebook"
-        registration_notification: "Registration Notifications" # TODO: Translate
-        registration_notifications: # TODO: Translate
-          digest: "One Email per Day"
-          immediate: "Immediately"
-          disabled: "No Notifications"
+        registration_notification: "Anmeldung Benachrichtigungen"
+        registration_notifications:
+          digest: "Einmal pro Tag"
+          immediate: "sofort"
```
Comment (Ardnived) [inline config/locales/de/models.yml:229]: This is a title, it should be capitalised.

File: config/locales/de/models.yml [hunk 6]
```diff
@@ -223,19 +223,19 @@ de:
...
+          disabled: "keine Benachrichtigungen"
```
Comment (Ardnived) [inline config/locales/de/models.yml:230]: This is a title, it should be capitalised.

File: config/locales/de/models.yml [hunk 7]
```diff
@@ -308,21 +308,21 @@ de:
...
-      client: # TODO: Translate
+      client:
         label: "Label"
         config: "Konfiguration"
         domain: "Domain"
         secret_key: "Geheime API-Schlüssel"
         public_key: "Public API-Schlüssel"
         enabled: "Freigegeben"
-        location: "Canonical Location"
+        location: "autorisierte Stelle"
```
Comment (Ardnived) [inline config/locales/de/models.yml:318]: This is a title, it should be capitalised.

**Review by Ardnived — CHANGES_REQUESTED** [review]
Looks good. Thanks for updating all this. I've pointed out a few places where I think the capitalisation might be wrong.

File: config/locales/de/models.yml (hunk 2 above)
Comment (mxr0) [reply config/locales/de/models.yml:85]: no, that is correct. welcome to the German Language :)
https://dict.leo.org/englisch-deutsch/responsibility
https://dict.leo.org/englisch-deutsch/responsibilities

File: config/locales/de/models.yml (hunk 3 above)
Comment (mxr0) [reply config/locales/de/models.yml:55]: ok, will keel online in small letters

File: config/locales/de/models.yml (hunk 4 above)
Comment (mxr0) [reply config/locales/de/models.yml:62]: ok. will change to "Vor-Ort-Veranstaltung" as that is what duden suggests.

File: config/locales/de/models.yml (hunk 5 above)
Comment (mxr0) [reply config/locales/de/models.yml:229]: will be Sofort

File: config/locales/de/models.yml (hunk 6 above)
Comment (mxr0) [reply config/locales/de/models.yml:230]: will be "Keine Benachrichtigungen"

File: config/locales/de/cms.yml (hunk 1 above)
Comment (mxr0) [reply config/locales/de/cms.yml:102]: can be "Unbenannt"

File: config/locales/de/models.yml (hunk 7 above)
Comment (mxr0) [reply config/locales/de/models.yml:318]: "Autorisierte Stelle"

**Comment by mxr0** [comment]
ok, incorporated the feedback.

**Pull request merged**